### PR TITLE
Add known issue CRD terminating in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/25178

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -764,7 +764,30 @@ When you have a configuration policy that is configured with `mustnothave` for t
...
 
 * link:https://github.com/stolostron/policy-collection/blob/main/stable/SI-System-and-Information-Integrity/policy-imagemanifestvuln.yaml[{product-title-short} policy for the Quay Container Security Operator]
-* link:https://github.com/stolostron/policy-collection/blob/main/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml[{product-title-short} policy for the Compliance Operator] 
+* link:https://github.com/stolostron/policy-collection/blob/main/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml[{product-title-short} policy for the Compliance Operator]
+
+[#configurationpolicy-crd-terminating]
+=== ConfigurationPolicy CRD is stuck in terminating
+
+When you remove the `config-policy-controller` addon from a managed cluster by disabling the policy controller in the `KlusterletAddonConfig` or by detaching the cluster, the `ConfigurationPolicy` CRD might get stuck in a terminating state. If the `ConfigurationPolicy` CRD is stuck in a terminating state, new policies might not be added to the cluster if the addon is re-installed later. You can also receive the following error:
```
Comment (cadawson) [inline release_notes/known_issues.adoc]: Add-on?

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -764,7 +764,30 @@ When you have a configuration policy that is configured with `mustnothave` for t
...
+
+----
+NonCompliant; Failed to create policy template create not allowed while custom resource definition is terminating
```
Comment (cadawson) [inline release_notes/known_issues.adoc]: Not you, but wow.

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -764,7 +764,30 @@ When you have a configuration policy that is configured with `mustnothave` for t
...
+----
+
+Use the following command to check if the CRD is stuck: 
+
+----
+oc get crd configurationpolicies.policy.open-cluster-management.io -o=jsonpath='{.metadata.deletionTimestamp}'
+----
+
+If a deletion timestamp is present on the resource, the CRD is stuck. To resolve the issue, remove all finalizers from configuration policies that remain on the cluster. Use the following command on the managed cluster and replace `<cluster-namespace>` with the managed cluster namespace:
```
Comment (cadawson) [inline release_notes/known_issues.adoc]: Do we need "present" here?

File: release_notes/known_issues.adoc [hunk 4]
```diff
@@ -764,7 +764,30 @@ When you have a configuration policy that is configured with `mustnothave` for t
...
+
+----
+oc get configurationpolicy -n <cluster-namespace> -o name | xargs oc patch -n <cluster-namespace> --type=merge -p '{"metadata":{"finalizers": []}}'
+----
+
+The configuration policy resources are automatically removed from the cluster and the CRD will exit its terminating state. If the addon has already been re-installed, the CRD will be recreated automatically without a deletion timestamp.
```
Comment (cadawson) [inline release_notes/known_issues.adoc]: "exits", rather than "will exit"?
"CRD is recreated", rather than "will be recreated"?
re-installed > reinstalled

**Review by cadawson — CHANGES_REQUESTED** [review]
Looks good. A couple of suggestions and updates. Thanks!

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (oafischer) [reply release_notes/known_issues.adoc]: Yeah...

@JustinKuli Can you confirm that is the correct error message?

Comment (JustinKuli) [reply release_notes/known_issues.adoc]: Sorry! I might have copied it from the wrong place... The message is
```
template-error; Failed to create policy template: create not allowed while custom resource definition is terminating
```

As seen below.

Screen Shot 2022-08-26 at 1 35 54 PM

What is "wow" about this, anything we can improve in the future?

Comment (oafischer) [reply release_notes/known_issues.adoc]: Thanks @JustinKuli, I've updated the message.

**Review by cadawson — APPROVED** [review]
Thanks! Looks good!

**Pull request merged**